docs/content/tutorial/resize-cluster.md

```diff
@@ -15,9 +15,19 @@ Let's dive in.
 
 Memory and CPU resources are an important component for vertically scaling your Postgres cluster. Couple with [tweaks to your Postgres configuration file]({{< relref "./customize-cluster.md" >}}), allowing your cluster to have more memory and CPU allotted to it can help it to perform better under load.
 
-It's important for instances in the same high availability set to have the same resources. PGO  lets you adjust CPU and memory within the `spec.instances.resources` section of the `postgresclusters.postgres-operator.crunchydata.com` custom resource. The layout of `spec.instances.resources` should be familiar: it follows the same pattern as the standard Kubernetes structure for setting [container resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
+It's important for instances in the same high availability set to have the same resources. PGO lets you adjust CPU and memory within the `resources` sections of the `postgresclusters.postgres-operator.crunchydata.com` custom resource. These include:
 
-For example, let's say we want to update our `hippo` Postgres cluster so that each instance has a limit of `2.0` CPUs and `4Gi` of memory. We can make the following changes to the manifest:
+- `spec.instances.resources` section, which sets the resource values for the PostgreSQL container, as well as any init containers in the associated pod.
+- `spec.instances.sidecars.replicacertcopy.resources` section, which sets the resources for the `replica-cert-copy` sidecar container.
+- `spec.instances.sidecars.exporter.resources` section, which sets the resources for the `exporter` sidecar container.
+- `spec.backups.pgbackrest.repoHost.resources` section, which sets the resources for the pgBackRest repo host container, as well as any init containers in the associated pod.
+- `spec.backups.pgbackrest.sidecars.pgbackrest.resources` section, which sets the resources for the `pgbackrest` sidecar container.
+- `spec.proxy.pgBouncer.resources` section, which sets the resources for the `pgbouncer` container.
+- `spec.proxy.pgBouncer.sidecars.pgbouncerconfig.resources` section, which sets the resources for the `pgbouncer-config` sidecar container.
```

Comment:
🤔 I go back and forth on what field names make the most sense to me here. For this one, I think I'd want to eliminate the stutter and just call it `config`: `spec.proxy.pgBouncer.sidecars.config.resources`.

The stutter in the pgBackRest sidecar name makes me think we should call it ~~`postgres` or~~ `instance` because that is where the sidecar runs.

`replicacertcopy` seems very specific to our current implementation, but it's accurate. I think we can manage deprecating it or renaming it in the controller in the future.

Reply:
I'm hesitant to change `pgbouncerConfig` to `config` if only because I'd like to keep the names as close as possible to the actual container names. In the event we add more containers, it should always be abundantly clear with sidecar we're referring to, independent of documentation. The same idea applies to the instance `pgbackrest` sidecar.

:thinking: Unless you're saying we should rename the actual containers?